### PR TITLE
feat: allow signature in the editor directly [CW-2409][CW-2352]

### PR DESCRIPTION
**Pull request by scmmishra**
## ToDo

- [x] Handle case when no signature is present
- [ ] Move signature utils to `@chatwoot/utils`

### CODE REVIEW

File: app/javascript/dashboard/helper/editorHelper.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+/**
```
Comment (pranavrajs) [inline app/javascript/dashboard/helper/editorHelper.js:1]: Should we move these to @chatwoot/utils or should we move @chatwoot/utils to here and make this a mono repo? 

Have a consistent utils with documentation pattern would help in mobile app as well.

Comment (scmmishra) [reply app/javascript/dashboard/helper/editorHelper.js:1]: Sounds good, I can move this to @chatwoot/utils for now. 

Making this a mono-repo will require some setup. We perhaps take it up later.

File: app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
           <woot-message-editor
             id="message-signature-input"
             v-model="messageSignature"
-            class="message-editor"
+            class="message-editor signature-editor"
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue:18]: Can we use it as an inline

```suggestion
            class="message-editor h-[10rem]"
```

File: app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue [hunk 3]
```diff
@@ -110,10 +110,8 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.profile--settings--row {
-  .ProseMirror-woot-style {
-    @apply h-20;
-  }
+.signature-editor {
+  @apply h-[10rem];
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue:114]:
```suggestion
.signature-editor {
  @apply h-[10rem];
```

**Review by iamsivin — COMMENTED** [review]
LGTM

**Review by iamsivin — APPROVED** [review]

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**